### PR TITLE
RCAL-180 Added support for dq_init step in cal_step.

### PR DESCRIPTION
**Pull request by PaulHuwe**
Added support for dq_init step (cal_step and RAD version dependency). 

Small tweak to mk_flat_file for ease of use.

### CODE REVIEW

**Comment by PaulHuwe** [comment]
Setup.cfg is temporarily pointed to the PR version of RAD.

**Review by ddavis-stsci — APPROVED** [review]
LGTM

**Comment by PaulHuwe** [comment]
I found an error and am patching it.

**Review by eslavich — COMMENTED** [review]
Looks good, just needs a different romanad requirement.

**Review by nden — APPROVED** [review]

File: setup.cfg [hunk 1]
```diff
@@ -26,7 +26,8 @@ install_requires =
     psutil>=5.7.2
     numpy>=1.16
     astropy>=4.0
-    romanad==0.4.0
+    # romanad>=0.4.0
+    romanad @ git+https://github.com/PaulHuwe/rad.git@RAD-25_DQInit_Calstep
```
Comment (nden) [inline setup.cfg:30]: this is what's breaking the tests. Wasn't the PR on RAD merged? If so, update to rad/master so that checks pass here

Comment (PaulHuwe) [reply setup.cfg]: Explicitly pointed to main now, and the tests pass.

**Pull request merged**